.gitignore

```diff
@@ -83,6 +83,7 @@ yarn.lock
 
 #IDE Files
 .vscode
+.idx
```

Comment:
why this ?

Reply:
I'm using a new IDE called [Project IDX](https://developers.google.com/idx). Comes with its own config directory, just like `.vscode`.

Reply:
ok make sense, can you raise this as a separate issue and pr and keep this pr separate . so we can move forward this qucikly